### PR TITLE
Test implementation with JSON API Resources

### PR DESCRIPTION
**Pull request by stravid**

### CODE REVIEW

File: test/request_helper.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      ActiveRecord::Base.connection.execute("TRUNCATE #{table}") unless skipped.include?(table)
+    end
+  end
+
+  # include Rack::Test::Methods
+  # include TransactionalDatabaseTestRuns
+
+  # def app
+    # Hanami::Container.new
+  # end
```
Comment (tombu) [inline test/request_helper.rb]: Looks like some legacy copy&paste stuff … ;)

**Comment by tombu** [comment]
I like it (using jsonapi-resources). Let´s give it a try to see how far we can take it :)

**Comment by tombu** [comment]
When testing the API with the frontend there was an issue like described here: https://github.com/cerebris/jsonapi-resources/issues/670

A proposed solution for this is to fake it.
```
def contact_person_id=(id)
  # this helper fixes a bug in jsonapi-resources 0.7.1.beta1
  # https://github.com/cerebris/jsonapi-resources/issues/670
end
````

Maybe there is a solution to fix it in the frontend – I just didn´t find one.

**Comment by stravid** [comment]
@tombu both issues should be resolved now.

**Pull request merged**